Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift

```diff
@@ -373,7 +378,7 @@ public final class SwiftTargetBuildDescription {
 
         let content =
             """
-            \(self.toolsVersion < .vNext ? "import" : "@_implementationOnly import") Foundation
+            import Foundation
```

Comment:
Removing this since it is not a thing that can work, a non-resilient package cannot hide its reliance on Foundation (or any other module).

Reply:
cc @ephemer